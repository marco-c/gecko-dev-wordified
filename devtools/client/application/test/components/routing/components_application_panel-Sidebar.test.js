/
*
Any
copyright
is
dedicated
to
the
Public
Domain
.
http
:
/
/
creativecommons
.
org
/
publicdomain
/
zero
/
1
.
0
/
*
/
"
use
strict
"
;
/
/
Import
libs
const
{
shallow
}
=
require
(
"
enzyme
"
)
;
const
{
createFactory
}
=
require
(
"
react
"
)
;
const
Sidebar
=
createFactory
(
require
(
"
devtools
/
client
/
application
/
src
/
components
/
routing
/
Sidebar
"
)
)
;
/
*
*
*
Test
for
Sidebar
.
js
component
*
/
describe
(
"
Sidebar
"
(
)
=
>
{
it
(
"
renders
the
expected
snapshot
"
(
)
=
>
{
const
wrapper
=
shallow
(
Sidebar
(
)
)
;
expect
(
wrapper
)
.
toMatchSnapshot
(
)
;
}
)
;
}
)
;
