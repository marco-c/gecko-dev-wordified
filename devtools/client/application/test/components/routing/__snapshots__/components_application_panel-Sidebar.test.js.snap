/
/
Jest
Snapshot
v1
https
:
/
/
goo
.
gl
/
fbAQLP
exports
[
Sidebar
renders
the
expected
snapshot
1
]
=
<
aside
className
=
"
sidebar
"
>
<
ul
className
=
"
sidebar__list
"
>
<
SidebarItem
key
=
"
sidebar
-
item
-
manifest
"
page
=
"
manifest
"
/
>
<
SidebarItem
key
=
"
sidebar
-
item
-
service
-
workers
"
page
=
"
service
-
workers
"
/
>
<
/
ul
>
<
/
aside
>
;
